ocs_ci/utility/utils.py

```diff
@@ -954,6 +954,34 @@ def ocsci_log_path():
     )
 
 
+def get_testrun_name():
+    """
+    Prepare testrun ID for Polarion (and other reports).
+
+    Return config.REPORTING["polarion"]["testrun_id"], if configured.
+    Otherwise prepare testrun ID based on Upstream/Downstream information,
+    OCS version and used markers.
+
+    Returns:
+        str: String containing testrun ID
+    """
```

Comment:
Leave empty line between Returns and """
eg: 
```
Returns:
        str: String containing testrun ID

"""
```

Reply:
Fixed in b123b44.